src/XR.tsx

```diff
@@ -144,7 +144,7 @@ export function VRCanvas({ children, ...rest }: ContainerProps) {
 
 export function ARCanvas({ children, sessionInit, ...rest }: ContainerProps & { sessionInit?: any }) {
   return (
-    <XRCanvas onCreated={({ gl }) => void document.body.appendChild(ARButton.createButton(gl, sessionInit))} {...rest}>
+    <XRCanvas onCreated={({ gl }) => void document.getElementById("ar-btn").appendChild(ARButton.createButton(gl, sessionInit))} {...rest}>
```

Comment:
This fights against React's architecture, which is a bit of a non-starter. If you want to support custom targets, try [forwarding refs](https://reactjs.org/docs/forwarding-refs.html).

I think appending to the body is a safe default though.